span-normalizer/raw-span-constants/src/main/proto/org/hypertrace/core/span/constants/v1/span_attribute.proto

```diff
@@ -23,6 +23,12 @@ enum Tcp {
     TCP_PORT = 2 [(string_value) = "tcp.port"];
 }
 
+// Authentication related attributes
+enum Auth {
```

Comment:
Now, we don't add in proto. We directly use java constants.
Where are we planning to use? In enricher?

Reply:
In api-anomaly detector and trainer